Sources/KlaviyoLocation/GeofenceService.swift

```diff
@@ -46,14 +46,31 @@ struct GeofenceService: GeofenceServiceProvider {
 
     func parseGeofences(from data: Data, companyId: String) throws -> Set<Geofence> {
         let response = try JSONDecoder().decode(GeofenceJSONResponse.self, from: data)
-        return try Set(response.data.map { rawGeofence in
-            try Geofence(
-                id: "_k:\(companyId):\(rawGeofence.id)",
-                longitude: rawGeofence.attributes.longitude,
-                latitude: rawGeofence.attributes.latitude,
-                radius: rawGeofence.attributes.radius
-            )
-        })
+        var geofences: Set<Geofence> = []
```

Comment:
Also, general question about this logic, when would there be a case where some geofences are pass parsing whereas other wont? 

FWIW, I think what you are doing here is legit but asking to see if you are already seeing something to made you add this code.